src/DynamoCoreWpf/UI/GuidedTour/CustomRichTextBox.cs

```diff
@@ -69,113 +72,124 @@ private static FlowDocument GetCustomDocument(string Text)
             string imageName = string.Empty;
             string hyperlinkName = string.Empty;
 
-            //This iteration is just for words in a paragraph if the text provided has several paragraphs then an additional foreach cycle needs to be added for iterating paragraphs
-            foreach (string word in Text.Split(' ').ToList())
+            //This will iterate each paragraph ()
+            foreach (string paragraph in paragraphsList)
             {
-                //A format for bold text was found
-                if (word.Trim().StartsWith("**"))
+                Paragraph para = new Paragraph()
                 {
-                    bBoldActive = true;
-                }
+                    Margin = new Thickness(0) // remove indent between paragraphs
+                };
+                BlockUIContainer imageContainer = new BlockUIContainer();
+                List bulletedItemsList = null;
 
-                //A format for inserting a image between text was found
-                if (word.StartsWith("%"))
+                //This iteration is just for words in a paragraph if the text provided has several paragraphs 
+                foreach (string word in paragraph.Split(' ').ToList())
                 {
-                    bImageActive = true;
-                }
+                    //A format for bold text was found
+                    if (word.Trim().StartsWith("**"))
+                    {
+                        bBoldActive = true;
+                    }
 
-                //A format for inserting bullet points was found (consider that for now we are just supporting bullet points ONLY at the end of the text)
-                if (word.StartsWith("-"))
-                {
-                    if (bulletedItemsList == null && bBulletListActive == false)
-                        bulletedItemsList = new List();
+                    //A format for inserting a image between text was found
+                    if (word.StartsWith("%"))
+                    {
+                        bImageActive = true;
+                    }
 
-                    bBulletListActive = true;
-                }
+                    //A format for inserting bullet points was found (consider that for now we are just supporting bullet points ONLY at the end of the text)
```

Comment:
I think you are adding support so that bullets in the middle of the text also display, right? Should we update the comments as well?

Reply:
@QilongTang updated in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12250/commits/c3995980180bb91f906ebbe6eb891924d0a584ea